Lib/opcode.py

```diff
@@ -45,8 +39,7 @@
 def is_pseudo(op):
     return op >= MIN_PSEUDO_OPCODE and op <= MAX_PSEUDO_OPCODE
 
-oplists = [hasarg, hasconst, hasname, hasjrel, hasjabs,
-           haslocal, hascompare, hasfree, hasexc]
+oplists = [hascompare, hasfree, hasexc]
```

Comment:
Maybe it's time to add a comment explaining what oplists actually means?

Reply:
oplists will hopefully be removed altogether by the time we're done with this.